code/modules/modular_computers/file_system/programs/jobmanagement.dm

```diff
@@ -24,10 +24,10 @@ GLOBAL_VAR_INIT(time_last_changed_position, 0)
 		"Head of Security",
 		"Chief Engineer",
 		"Research Director",
-		"Security Sergeant",
+		"Security Sergeant", //SKYRAT EDIT CHANGE START
 		"Security Medic",
 		"Clown",
-		"Blueshield",
+		"Blueshield", //SKYRAT EDIT END
 		"Chief Medical Officer",
 		"Quartermaster",
 		"Shuttle Pilot")
```

Comment:
Shuttle pilot should be in the skyrat edit (maybe QM? Not actually sure on that)